package.json

```diff
@@ -19,8 +19,9 @@
     "graphql-ruby-client": "^1.4.0",
     "graphql-tag": "^2.9.2",
     "hoist-non-react-statics": "^3.0.1",
-    "numeral": "^2.0.6",
+    "jscodeshift": "^0.6.4",
```

Comment:
Just noting that I'm personally a fan of making the addition of a lib its own commit (as it's done here). Other devs feel like adding the lib should be in the same commit as when it's used, but this way makes it _much_ easier for me to review. (Also, for other kinds of projects like `react-native`, it is _very_ useful to have libs added in their own commits).

Reply:
I fall into the latter camp preferring to add the library with its usage. Not requesting a change, but curious how the split is beneficial in `react-native`.

Reply:
Hrm. This should be under devDependencies instead.